.github/workflows/comment_on_first_contribution.yml

```
@@ -0,0 +1,56 @@
+name: Comment on first contribution
+
+on:
+  workflow_call:
+    inputs:
+      pull_request_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          pull request.
+        type: string
+        required: false
+      issue_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          issue.
+        type: string
+        required: false
+
+jobs:
+  comment:
+    if: >
+      (
+        (github.event_name == 'pull_request_target' &&
+         inputs.pull_request_message) ||
+        (github.event_name == 'issues' && inputs.issue_message)
+      ) &&
```

I think this would read a lot easier if you remove one line break. We don’t enforce a maximum line length on YAML.

```suggestion
      (
        (github.event_name == 'pull_request_target' && inputs.pull_request_message) ||
        (github.event_name == 'issues' && inputs.issue_message)
      ) &&
```